openslides_backend/action/actions/user/password_mixin.py

```diff
@@ -0,0 +1,27 @@
+import random
+from typing import Any, Dict
+
+from ...action import BaseAction
+
+
+class PasswordCreateMixin(BaseAction):
+    def set_password(self, instance: Dict[str, Any]) -> Dict[str, Any]:
+        password = instance.get("default_password", "")
+        instance["default_password"] = password
```

Comment:
This is redundant

Reply:
Done

Reply:
No, I meant specifically the two marked lines, because you are retrieving the field from `instance`and then directly writing it back :D

Reply:
Ah, I see you already resolve it yourself